### ISSUE TITLE
Carousel Accessibility: navigation dots

### ISSUE THREAD

**Issue by digilou** [FormidableLabs/nuka-carousel#780]
# Bugs and Questions

## Prerequisites

**Feel free to delete this section if you have checked off all of the following.**

- [x] I've searched open [issues](https://github.com/FormidableLabs/nuka-carousel/issues) to make sure I'm not opening a duplicate issue
- [x] I have read through the docs before asking a question
- [x] I am using the latest version of nuka-carousel

## Describe Your Environment

- What version of nuka-carousel are you using? 4.7.5
- What version of React are you using? 16.13.1
- What browser are you using? Firefox 88.0.1

## Describe the Problem

The [navigation dots](https://github.com/FormidableLabs/nuka-carousel/blob/5812a3ca6609e6d7db021f6abcf69ef21bb5d7e5/src/default-controls.js#L244) are using `svg` for display, but this element doesn't have alternate text and isn't hidden from screen readers. Screen reader users may wonder what is the SVG graphics' purpose.

An `aria-label` is assigned to each parent`button`, one good approach would be to hide the `svg` from screen readers with `aria-hidden`. Adding `focusable=false` is a common solution for an IE bug. It prevents screen reader users from focusing on the button then the SVG. Approximately [11% of screen reader users still use IE11](https://webaim.org/projects/screenreadersurvey8/#devices#browsers) because it pairs best with their screen reader software.
```
<svg aria-hidden="true" focusable="false" className="paging-dot" width="6" height="6>
    <circle cx="3" cy="3" r="3" style="fill: black;"></circle>
</svg>
```

## Additional Information
This issue pertains to WCAG 2, Level A: [success criterion 1.1.1](https://www.w3.org/WAI/WCAG21/quickref/#non-text-content).
